media-libs/assimp/assimp-5.2.5.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Importer library to import assets from 3D files"
+HOMEPAGE="https://github.com/assimp/assimp"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/5.2.4"
```

Comment:
Unfortunately it looks like this has to be bumped.
```
 FILES:-usr/lib64/libassimp.so.5.2.4
 FILES:+usr/lib64/libassimp.so.5.2.5
   ABI: libassimp.so.5(64) func(+105,-55) vars(+4) [BREAKING]
```

Reply:
Good catch. Must have overseen this.